pyproject.toml

```diff
@@ -1,11 +1,59 @@
 [build-system]
 requires = [
-    "setuptools>=42",
-    "setuptools_scm[toml]>=3.4",
-    "wheel",
-    "milksnake>=0.1.2"
+    "maturin>=0.13,<0.14",
```

Comment:
Um, is there any reason to limit the versions here?
Upstream maturin is already at 0.14.6 and that is also what we package on Arch Linux.

Reply:
I originally had some issue with `maturin` 0.14 (it might have been that it requires Rust 2021 Edition), but once things were working I should have circled back and tried again.